jwcrypto/jwa.py

```diff
@@ -44,6 +56,22 @@ def algorithm_use(self):
         raise NotImplementedError
 
 
+# Note: l is the number of bits, which should be a multiple of 16
+def _encode_int(n, l):
```

Comment:
This code is just moved, please open an issue for problems with the original code